src/layerstack.stanza

```diff
@@ -214,8 +220,8 @@ public val SoldermaskMaterial = DielectricMaterial(
   is-soldermask = true
 )
 
-public defstruct LayerSpec:
-  name:Maybe<String>
+public defstruct LayerSpec <: Equalable :
+  name:Maybe<String> with: (setter => set-name)
```

Comment:
Let's make this internal setter `set-name?` where `?` indicates maybe. 

Then lets create a `defn set-name` without the `?` that takes just a `String` argument and converts it to `Maybe<String>` with `One`.

That way the user facing interface is simpler.

Reply:
Should we change the field name `name` to `name?` to be consistent?
Then, add a function `name` to access it as a String or "" if empty.

Reply:
I don't have an issue with that proposal as long as we stay consistent. So anywhere in this file (or ideally, anywhere in JSL) where we use a `Maybe` needs to use a `?` suffix.

Reply:
On second thought, not changing the field name `name` to `name?`.
There are many other fields with the `Maybe` type. To be consistent, we would need to change all of them to `field?`. Maybe you don't want so many changes.